docs/reference-metrics-opentelemetry.md

```diff
@@ -402,3 +402,26 @@ The number of pending topic load operations in the broker. When it reaches thres
 The maximum number of pending topic load operations in the broker. Equal to "maxConcurrentTopicLoadRequest" defined in broker.conf.
 * Type: UpDownCounter
 * Unit: `{operation}`
+
+### Web Executor Service metrics
+
+#### pulsar.web.executor.thread.limit
```

Comment:
Aren't we missing the scope? `pulsar.broker...` ? Or they are for both client and proxy and function worker?

Reply:
Yes, these would be the same metrics across the other components.